pkg/controller/compliancescan/aggregator.go

```diff
@@ -139,7 +140,6 @@ func (r *ReconcileComplianceScan) deleteAggregator(instance *compv1alpha1.Compli
 
 func isAggregatorRunning(r *ReconcileComplianceScan, scanInstance *compv1alpha1.ComplianceScan, logger logr.Logger) (bool, error) {
 	logger.Info("Checking aggregator pod for scan", "ComplianceScan.Name", scanInstance.Name)
-
 	podName := getAggregatorPodName(scanInstance.Name)
-	return isPodRunning(r, podName, common.GetComplianceOperatorNamespace(), logger)
+	return isPodRunning(r, podName, common.GetComplianceOperatorNamespace(), time.Duration(0), logger)
```

Comment:
is the meaning of time.Duration(0) "no timeout"? In that case we might want to add a global variable with that name for better readability